src/pages/quickstart/NavigationOverlay/NavigationFlow.tsx

```diff
@@ -48,25 +48,31 @@ const navigationOptions: NavigationOption[] = [
     id: 'delegation-toolkit-1',
     title: 'I want to create a smart account',
     description: 'Delegation Toolkit',
-    link: '/delegation-toolkit/development/get-started/smart-account-quickstart',
+    link: '/delegation-toolkit/get-started/smart-account-quickstart/',
+  },
+  {
+    id: 'delegation-toolkit-4',
+    title: 'I want to perform executions on behalf of smart accounts',
+    description: 'Delegation Toolkit',
+    link: '/delegation-toolkit/guides/delegation/execute-on-smart-accounts-behalf',
   },
   {
     id: 'delegation-toolkit-2',
-    title: 'I want to send gasless transactions',
+    title: "I want to perform executions on behalf of MetaMask users'",
```

Comment:
```suggestion
    title: 'I want to perform executions on behalf of MetaMask users',
```

("users" doesn't need an apostrophe)